gunicorn/sock.py

```diff
@@ -53,7 +53,7 @@ def set_options(self, sock, bound=False):
     def bind(self, sock):
         sock.bind(self.cfg_addr)
 
-    def close(self):
+    def close(self, locked=False):
```

Comment:
this is for consistency since the close function will be call with an arg anyway. Maybe it could be `**kwargs`?